draft-mouris-cfrg-mastic.md

```diff
@@ -399,9 +399,8 @@ Mastic uses a combination of techniques to certify the validity of this output.
    as specified in {{vidpf-construction}} does not as specified permit
    evaluation at the root of the VIDPF tree. Instead, each Aggregator
    computes a share of `beta` by evaluating the VIDPF tree at prefixes `0`
-   and `1` and `level == 0` and adding them up. One-hot Verifiability and
-   Path Verifiability imply that the sum is equal to the Aggregator's share
-   of `beta`.
+   and `1` and `level == 0` and adding them up. One-hot Verifiability implies
+   that the sum is equal to the Aggregator's share of `beta`.
```

Comment:
This one is a bit tricky as one-hotness alone implies only that the sum is non-zero at one position. We need both properties together to be sure that this non-zero value is indeed `beta` (which we have checked at the root with the FLP).